ci_framework/roles/repo_setup/tasks/artifacts.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: Get DLRN hash
-  ansible.builtin.get_url:
-    url: "{{ cifwm_repo_setup_dlrn_uri }}{{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }}-{{ cifmw_repo_setup_branch }}/{{ cifmw_repo_setup_promotion }}/delorean.repo.md5"
+- name: Run repo-setup=get-hash
+  ansible.builtin.shell: |
+    {{ cifmw_repo_setup_basedir }}/venv/bin/repo-setup-get-hash
+      --dlrn-url {{ cifwm_repo_setup_dlrn_uri }} \
+      --os-version {{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }} \
+      --tag cifmw_repo_setup_promotion }} \
+      --release {{ cifmw_repo_setup_branch }} \
+    {% if cifwm_repo_setup_component is defined %}
+      --component {{ cifmw_repo_setup_component }} \
+    {% endif %}
+      --json | jq -r '.full_hash'
```

Comment:
jq isn't installed by default, you want to add it in the bindep.txt I think

Reply:
Maybe we don't need to add additional packages but `community.general.json_query`

Reply:
right, indeed.

Reply:
replaced with community.general.json_query but we need to install python3-jmespath in order to get json_query working.

Reply:
Oh, you are right, I've seen now. We should add things anyway.